### PR TITLE
AddingPowersupport_marathon-client

### PR DESCRIPTION
**Pull request by santosh653**
creating marathon-client.travis.yml with proposed changes.,

### CODE REVIEW

**Comment by santosh653** [comment]
@Siddhesh-Ghadi Can you please review this one?

**Comment by santosh653** [comment]
@Siddhesh-Ghadi @gururajrkatti Could you please review this PR?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]
Pl. mention the comment in the travis file mentioning the reason for excluding jdk8 from amd64.
Also oraclejdk11 not supported on ppc64le. Is it required ?

**Comment by gururajrkatti** [comment]
@Siddhesh-Ghadi , request you to confirm on this

**Comment by Siddhesh-Ghadi** [comment]
Please `exclude: oraclejdk11` on power as it is not supported and also add comment stating the same. @santosh653

**Comment by santosh653** [comment]
@Siddhesh-Ghadi I have done the needful, please check once & confirm.,

File: travis-ymls/marathon-client.travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Package             : marathon-client
+# Source Repo         : https://github.com/willroden/marathon-client
+# Travis Job Link     : https://travis-ci.com/github/santosh653/marathon-client
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+# oraclejdk7/8/11 not supported on power hence running the code with latest openjdk11 on both Intel & Power to make it consistent.
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/marathon-client.travis.yml:12]: exclude oracle11 only on ppc64le.

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

File: travis-ymls/marathon-client.travis.yml (hunk 1 above)
Comment (santosh653) [reply travis-ymls/marathon-client.travis.yml]: @Siddhesh-Ghadi Kindly verify now if the changes are correct? I have modified as per your comments.

Comment (Siddhesh-Ghadi) [reply travis-ymls/marathon-client.travis.yml]: Please update the travis-job link with latest job. @santosh653

Comment (santosh653) [reply travis-ymls/marathon-client.travis.yml]: @Siddhesh-Ghadi I have updated the Travis-job link as suggested by you. Kindly let me know if it is all right now.
Thanks !!

File: travis-ymls/marathon-client.travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : marathon-client
+# Source Repo         : https://github.com/willroden/marathon-client
+# Travis Job Link     : https://travis-ci.com/github/santosh653/marathon-client
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/marathon-client.travis.yml:5]: Please update this link with latest build. @santosh653. Also squash the commits.

**Comment by santosh653** [comment]
Done Kindly check.,

**Comment by santosh653** [comment]
@seth-priya @gururajrkatti I have done the needful Can we have this merged?

**Comment by Siddhesh-Ghadi** [comment]
Still pointing to old job

```
arch:
  - amd64
  - ppc64le
language: java
jdk:
  - openjdk11
```
https://travis-ci.com/github/santosh653/marathon-client/builds/213465911/config

@santosh653

**Comment by santosh653** [comment]
@Siddhesh-Ghadi I have made changes to the forked package url:
( https://github.com/santosh653/marathon-client/blob/master/.travis.yml )
The Build is also going fine. Can you please approve this PR?

**Comment by Siddhesh-Ghadi** [comment]
Please squash the commits. Rest all looks good. @santosh653

**Comment by santosh653** [comment]
@Siddhesh-Ghadi Can you please approve now? I have squashed the commits.

**Comment by Siddhesh-Ghadi** [comment]
Ready for merge @seth-priya

**Pull request merged**